src/CommunityToolkit.Mvvm.SourceGenerators/ComponentModel/ObservablePropertyGenerator.Execute.cs

```diff
@@ -961,7 +961,7 @@ public static MemberDeclarationSyntax GetPropertySyntax(PropertyInfo propertyInf
                         .AddArgumentListArguments(
                             Argument(fieldExpression),
                             Argument(IdentifierName("value")))),
-                    Block(setterStatements.ToArray()));
+                    Block(setterStatements.AsEnumerable()));
```

Comment:
Did the allocation show up in traces?

Reply:
Not really, that wasted allocation was just bothering me 😄
And there's plenty more like that in other generators.

Reply:
Yup likely the impact is low here, but good to remove an unnecessary allocation anyway.